odoo/service/server.py

```diff
@@ -162,6 +162,17 @@ def end_headers(self, *a, **kw):
             self.rfile = BytesIO()
             self.wfile = BytesIO()
 
+    @property
+    def path(self):
```

Comment:
Add a comment:
`# patched property so that we add the request fragment which will be logged by werkzeug`